src/renate/benchmark/datasets/nlp_datasets.py

```diff
@@ -142,14 +142,13 @@ def tokenize_fn(batch):
 class MultiTextDataModule(DataIncrementalDataModule):
     """
     Inspired by the dataset used in "Episodic Memory in Lifelong Language Learning"
-    by d’Autume et al. this is a collection of five different datasets that we call domains:
+    by d’Autume et al. this is a collection of four different datasets that we call domains:
     AGNews, Yelp, DBPedia and Yahoo Answers.
 
     The output space if the union of the output space of all the domains.
     The dataset has 33 classes: 4 from AGNews, 5 from Yelp, 14 from DBPedia, and 10 from Yahoo.
 
     The maximum allowed size for the training set is 115000 and for the test set is 7600.
```

Comment:
What does 'allowed' mean here?

Reply:
anything larger will raise an exception. i assume the smallest dataset has a size of 115k train or 7600 test points.

any suggestion how to improve the description to explain it better?